server/routes/aws/cloudwatch_routes.py

```
@@ -0,0 +1,287 @@
+    # --- Notification ---
+    if sns_type not in (_SNS_NOTIFICATION, ""):
+        logger.info(
+            "[CLOUDWATCH] Unknown SNS type=%s for user %s, ignoring", sns_type, sanitize(user_id)
+        )
+        return jsonify({"received": True})
+
+    # SNS wraps the alarm JSON in a "Message" string field
+    raw_message = sns_message.get("Message") or ""
+    try:
+        payload = json.loads(raw_message) if raw_message else sns_message
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)